esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -2230,7 +2230,7 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor *udesc, co
         df->setRestrictedAccess(changeRestriction==CDFUChangeRestriction_Restrict);
     }
 
-    offset_t size=queryDistributedFileSystem().getSize(df), recordSize=df->queryAttributes().getPropInt64("@recordSize",0);
+    offset_t size=df->getFileSize(true, false), recordSize=df->queryAttributes().getPropInt64("@recordSize",0);
```

Comment:
@richardkchapman this isn't directly connected to removing the old code.  Although the code for the two calls looks identical, do you want to keep this change?

Reply:
It is required to remove the link to the dalift dll from esp services.